rpc/client.go

```diff
@@ -292,11 +314,22 @@ func (rc *rpcClient) PaymentChannelUpdatesChan(paymentChannelId types.Destinatio
 	return c
 }
 
-func waitForRequest[T serde.RequestPayload, U serde.ResponsePayload](rc *rpcClient, method serde.RequestMethod, requestData T) (U, error) {
+// unauthenticatedWaitForRequest calls waitForRequest with an empty auth token
+func unauthenticatedWaitForRequest[T serde.RequestPayload, U serde.ResponsePayload](rc *rpcClient, method serde.RequestMethod, requestData T) (U, error) {
+	return waitForRequest[T, U](rc, method, requestData, "")
+}
+
+// authenticatedWaitForRequest calls waitForRequest with an empty auth token
```

Comment:
I think there must be a typo on this line?

Reply:
Good catch. Fixed in cf18e248.